src/card.tsx

```diff
@@ -197,15 +283,15 @@ export const MediaCard = React.forwardRef<HTMLDivElement, MediaCardProps>(
         ref
     ) => {
         return (
-            <MaybeDismissable onClose={onClose} aria-label={ariaLabel}>
+            <MaybeWithActions onClose={onClose} actions={actions} aria-label={ariaLabel} isInverse>
                 <Boxed
                     className={styles.boxed}
                     dataAttributes={{'component-name': 'MediaCard', ...dataAttributes}}
                     ref={ref}
                     width="100%"
                     height="100%"
                 >
-                    <MaybeSection className={styles.mediaCard} aria-label={ariaLabel}>
+                    <section className={styles.mediaCard}>
```

Comment:
I think this should be just a `<div>` now, MaybeWithActions always renders a `<section>`, we don't want two nested sections